game-core/src/main/java/tools/map/making/PlacementPicker.java

```diff
@@ -464,7 +490,11 @@ public void paint(final Graphics g) {
             g.fillRect(item.x, item.y, placeWidth, placeHeight);
             if (showOverflowMode && !pointIter.hasNext()) {
               g.setColor(Color.gray);
-              g.fillRect(item.x + placeWidth, item.y + placeHeight / 2, placeWidth, 4);
+              if (currentOverflowToLeft) {
+                g.fillRect(item.x - placeWidth, item.y + placeHeight / 2, placeWidth, 4);
+              } else {
+                g.fillRect(item.x + placeWidth, item.y + placeHeight / 2, placeWidth, 4);
+              }
```

Comment:
Like in the first example, you can't spot the small difference if you look for a short amount of time.